controller/WebController.php

```diff
@@ -329,23 +329,47 @@ public function invokeGlobalSearch($request)
         $vocabObjects = array();
         if ($vocids) {
             foreach($vocids as $vocid) {
-                $vocabObjects[] = $this->model->getVocabulary($vocid);
+                try {
+                    $vocabObjects[] = $this->model->getVocabulary($vocid);
```

Comment:
How to trigger the problem addressed here:
http://localhost/Skosmos/fi/search?clang=fi&q=kissa&vocabs=ysa+yso+xyzzy

(here, xyzzy is a vocabulary ID not found in configuration)

Results in:
[Wed Sep 08 14:26:37.964415 2021] [php7:error] [pid 193585] [client 127.0.0.1:35836] PHP Fatal error:  Uncaught ValueError: Vocabulary id 'xyzzy' not found in configuration. in /var/www/html/Skosmos/model/Model.php:420\nStack trace:\n#0 /var/www/html/Skosmos/controller/WebController.php(332): Model->getVocabulary()\n#1 /var/www/html/Skosmos/index.php(44): WebController->invokeGlobalSearch()\n#2 {main}\n  thrown in /var/www/html/Skosmos/model/Model.php on line 420

Reply:
This fix may address #1170

Reply:
This change fixes one particular path that could lead to the error seen #1170 (and copied above), but there are probably many other ways of triggering the same ValueError.